src/views/sessions/SessionList/BaseList.vue

```diff
@@ -78,7 +82,9 @@ export default {
             formatter: DetailFormatter,
             formatterArgs: {
               drawer: true,
-              getTitle: ({ row }) => { return row.user },
+              getTitle: ({ row }) => {
+                return row.user
+              },
               getRoute: ({ row }) => {
                 return {
                   name: 'UserDetail',
```

Comment:
This is a template for creating Vue components using TypeScript or JavaScript. There doesn't seem to be any specific irregularities, potential issues, or optimizations suggestions provided in this text snippet. If you have further questions about TypeScript or Javascript programming concepts like classes, interfaces, constructors, methods and properties, please let me know!